assingment.playground/Contents.swift

```diff
@@ -2,5 +2,41 @@ import Foundation
 
 private let workplace = Workplace()
 
-workplace.setEmptySeat(count: <#Int#>)
-workplace.employeesInOffice(employeeCount: <#Int#>)
+workplace.setEmptySeat(count: 9)
+workplace.employeesInOffice(employeeCount: 6)
+
+public func isSeatAvailable(_ seatIndex: Int) -> Bool {
+    let seat = workplace.seats[seatIndex]
+    if seat == nil {
+        return true
+    } else {
+        return false
+    }
+}
+
+public func occupyEmptySeat(index: Int, employee: String = "👩🏻‍💼") {
+    if index >= 0 && index < workplace.seats.count {
+    if isSeatAvailable(index) {
+        workplace.seats[index] = employee
+    } else {
+        if let oldEmployee = workplace.seats[index] {
+            print("\(employee) tried to take seat number \(index + 1), unfortunatelly that seat was occupied by \(oldEmployee).")
+        }
+    }
+    } else {
+            print ("Something went wrong! index must be between 0 and \(workplace.seats.count - 1)!")
+    }
+}
+
+public func workIsLeftBy(employee: String) {
+    for index in 0 ... workplace.seats.count - 1 {
```

Comment:
Let's re-read the criteria:

> Write a method named `workIsLeftBy` that takes in a parameter named `employee` of type `String`. Method should iterate through currently present workers, and all employees that match `employee` should leave workplace making seats available. If there are no employees that match `employee` output a message **"`employee` seems to be out-of-office."**.

Therefore, we understand that before we start iterating through array of seats, we need to check [if there is a seat occupied by such `employee`](https://stackoverflow.com/questions/7378228/check-if-an-element-is-present-in-an-array). If there is we do the remaining logic - iterate and and ask employees to leave, otherwise, we print the error message.